test/unit/dropdown_spec.js

```diff
@@ -75,6 +75,34 @@ describe('Dropdown', function() {
     });
   });
 
+  describe('when instantiated with a custom empty template', function() {
+    beforeEach(function() {
+      this.view.destroy();
+      this.view = new Dropdown({
+        menu: this.$menu,
+        datasets: [{}],
+        templates: {
+          empty: '<h3 class="empty">this is empty</3>'
```

Comment:
Yes, I didn't manage to avoid that :/ I need to get a reference to the "empty" DOM node I'm adding to the menu so I can show/hide it -> and nesting it's content in a div was the only option I was thinking about.